Stratis.Bitcoin.Features.BlockStore.Tests/Stratis.Bitcoin.Features.BlockStore.Tests.csproj

```diff
@@ -22,13 +22,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Remove="Stratis.Bitcoin.BlockStore.Tests\**" />
-    <EmbeddedResource Remove="Stratis.Bitcoin.BlockStore.Tests\**" />
-    <None Remove="Stratis.Bitcoin.BlockStore.Tests\**" />
+    <Compile Remove="Stratis.Bitcoin.Features.BlockStore.Tests\**" />
+    <EmbeddedResource Remove="Stratis.Bitcoin.Features.BlockStore.Tests\**" />
+    <None Remove="Stratis.Bitcoin.Features.BlockStore.Tests\**" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="DBreeze" Version="1.85.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0-preview-20170618-03" />
```

Comment:
Can we please have the non-preview version?
We normally use preview versions only when necessary.

Reply:
Changed to use the same versions used in the Api Tests.